nginx.conf

```diff
@@ -0,0 +1,31 @@
+events { }
+
+
+http {
+    include mime.types;
+
+    gzip on;
+    gzip_disable "msie6";
+
+    gzip_vary on;
+    gzip_proxied any;
+    gzip_comp_level 6;
+    gzip_buffers 16 8k;
+    gzip_http_version 1.1;
+    gzip_min_length 1100;
+    gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript application/javascript application/vnd.ms-fontobject application/x-font-ttf font/opentype image/svg+xml image/x-icon;
+
+    server {
+        listen 80;
+        listen 443 ssl;
+        ssl_certificate ../ssl/localhost.crt;
```

Comment:
Have this error with SSL in chrome. I had this kind of error when I was doing this task myself. I don't remember how exactly - Google is for help, but it is 100% fixable.
![Uploading Screenshot 2021-05-04 at 22.44.41.png…]()